Content.Server/Abilities/Boxer/Boxer/BoxingGlovesComponent.cs

```diff
@@ -0,0 +1,13 @@
+using Content.Shared.Damage;
+
+namespace Content.Server.Abilities.Boxer
+{
+    /// <summary>
+    /// Boxer gets a bonus for these, and their fists, but not other unarmed weapons.
+    /// </summary>
+    [RegisterComponent]
+    public sealed class BoxingGlovesComponent : Component
+    {
+        public bool IsActive = false;
```

Comment:
This is never used in the system, normally `IsActive` serves as part of a check to make sure the piece of clothing is giving its effects to the wearer and remove said effects when they are not worn.

Reply:
yeah thats what it was originally, ancient pr thats been reworked a bunch moment